### ISSUE TITLE
Tooltip blinking when target has children nodes

### ISSUE THREAD

**Issue by batcer** [bsidelinger912/react-tooltip-lite#35]
The target button has the following HTML:
```
<div class="button button--round">
     <div class="icon icon--fullscreen"></div>
</div>
```
where `icon` has a background image with an icon. Icon div is smaller that the button itself.
When pointer passes over the button into the icon, tooltip dissappears. After pointer is moved a little bit again over the icon, tooltip appears again.

**Comment by bsidelinger912** [comment]
Hey @batcer this is interesting, I think I know why, but will need to tinker a bit more to confirm.  Will address this ASAP

**Comment by FIIFE** [comment]
Similar issue here, but in my case the target element is a button and it seems to cause an additional problem. Only the icon inside it triggers the tooltip.

`<Tooltip content='cant delete because'>
<button disabled><FontAwesomeIcon icon='trash-alt' /></button>
</Tooltip>`

**Comment by bsidelinger912** [comment]
@batcer and @FIIFE this issue should be fixed now with v1.9.2

**Comment by bsidelinger912** [comment]
Closing this now as I believe it's been fixed.
